vogue/commands/load/application_tag.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import click
+import json
+from vogue.load.application_tag import load_aplication_tags
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("apptag", short_help="Reads json with application tag info.")
+@click.option('-a', '--application-tags', required=True, help='json formatted application tags')
+@click.pass_context
+
+
+def application_tags(context, application_tags: list):
```

Comment:
`application_tags` can never be a list when it comes from the CLI. It should be `str` or when it's a file, you could add it as a file, e.g.:
```suggestion
def application_tags(context, application_tags):
```